### PR TITLE
freeze k/website before release

### PR DESCRIPTION
**Pull request by simplytunde**
https://github.com/kubernetes/sig-release/tree/master/release-team/role-handbooks/docs#release-week-week-12

The goal is to freeze k/website 24hrs before release.
🛑 This needs to be merged exactly on Tuesday Sep 17th 4pm PT 🛑

### CODE REVIEW

**Comment by alvaroaleman** [comment]
Thanks! Removing myself as a reviewer as this change is not related to Prow itself but the prow.k8s.io config.
/uncc

**Comment by simplytunde** [comment]
This needs to be merged Tuesday Sep 17th 4pm PT

**Comment by Katharine** [comment]
This is not what we usually mean by "staging repo", I think, but it will have the effect it looks like you want.

Could you add a comment linking to https://github.com/kubernetes/sig-release/tree/master/release-team/role-handbooks/docs#release-week-week-12 please?

File: prow/config.yaml [hunk 1]
```diff
@@ -390,6 +390,7 @@ tide:
     - kubernetes/sample-apiserver
     - kubernetes/sample-cli-plugin
     - kubernetes/sample-controller
+    - kubernetes/website
```
Comment (simplytunde) [inline prow/config.yaml]: https://github.com/kubernetes/sig-release/tree/master/release-team/role-handbooks/docs#release-week-week-12

**Pull request merged**